admin/installer/cloudformation.py

```diff
@@ -115,6 +116,23 @@ def create_cloudformation_template_options():
             MAX_CLUSTER_SIZE
         )
     )
+
+    parser.add_argument(
+        u'--client-ami-map-body',
+        type=json.loads,
+        help=u'A JSON map of AWS region to AMI ID for client.',
```

Comment:
It'd be nice to list for which regions you have to provide AMIs.

Is it all regions? If so that's really annoying.